### DISCUSSION TITLE
Problema dopo aggiornamento HA

### DISCUSSION THREAD

**Discussion by grane90** [indomus/forum#2154] [HUB: Home Assistant]
Salve. Avevo home assistant installato su docker su rp4 bello e funzionante con tanto di Supervised
Vedendo la notifica e curioso per le "tile" colori ecc, mi è venuta la bella idea di aggiornare ha core da riga di comando seguendo la guida di indomus. I comandi che ho lanciato sono stati:

- sudo docker stop homeassistant
- Sudo rm homeassistant 
- Sudo docker pull ghcr.io/home-assistant/home-assistant:stable

Poi naturalmente ho creato il nuovo container con:

- Sudo docker run --init -d --name="homeassistant" -e "TZ=Europe/Rome" -v "/home/pi/homeassistant:/config" -v "/run/dbus:/run/dbus:ro" --net=host --restart=always ghcr.io/home-assistant/home-assistant:stable

Risultato: non raggiungevo ha. Allora su portainer ho tolto la spunta su "init" ed è partito. Solamente che (e qui viene il guaio) non vedo più la "versione" di ha con supervised ma una versione "latest" di core con pochissime entità (quelle che probabilmente avevo prima di implementare supervised). 
Posto qualche riga di log di Supervised:

22-12-10 14:04:20 ERROR (MainThread) [supervisor.homeassistant.api] Error on call https://172.30.32.1:8123/api/config: Cannot connect to host 172.30.32.1:8123 ssl:False [[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:997)]

Ho provato a fare un restore da riga di comando dell'ultimo backup che avevo, ma nulla. Credo di aver fatto una porcata con docker, magari risolvibile banalmente magari no, questo me lo direte voi.

Scusate la lunghezza del post. Spero mi possiate aiutare. grazie

**Comment by snakuzzo** [comment]
Innanzi tutto ti consiglio di utilizzare `docker-compose` o `docker compose`.
Poi...utilizzi un reverse proxy oppure no ? Dall'errore presumo tu abbia configurato i certificati direttamente nel `configuration.yaml`, giusto ?

**Reply by grane90** [reply]
Grazie per la risposta.
Quindi dovrei provare a togliere il container e ricaricarlo sfruttando docker compose intendi?
Comunque si, mi ricordo che nel file configuration.yaml vecchio c'erano le impostazioni di let's encrypt.
Ora mi ritrovo con la cartella /home/pi/ homeassistant completamente vergine.
Probabilmente dico una boiata, se io copiassi e incollassi la cartella di Ha "vecchia" risolverei qualcosa?

**Reply by snakuzzo** [reply]
Aspetta ... Che vuol dire che hai la dir `/home/pi` vergine? Devi mappare la directory contenente i vecchi file

**Reply by grane90** [reply]
Credo sia proprio questo il problema. Da terminale mi accorgo che la cartella di home assistant "giusta" sarebbe in /usr/share/hassio/homeassistant . Mentre  il volume del container punta a /home/pi/homeassistant perché quando l'ho ricreato copiando il codice della guida, da imbecille, ho dimenticato di modificare la path corretta. 
Ora ho mappato la vecchia directory sul nuovo container, ho riavviato tutto, però da browser vedo ancora la dashboard che avevo prima del supervisor, anche entrando in incognito quindi senza cache. Devo fare qualcos'altro?

**Reply by snakuzzo** [reply]
Non ho capito cosa vedi
